tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md

```diff
@@ -109,49 +109,49 @@ The following sub-sections outline various scenarios in which the `custom-pallet
 
 Verify that the counter can be successfully incremented under normal conditions, ensuring the increment works and the correct event is emitted.
 
-```rust
+```rust title="tests.rs"
 --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/pallets/custom-pallet/src/tests.rs:61:81'
 ```
 
 ### Preventing Value Overflow
 
 Test that the pallet prevents incrementing beyond the maximum allowed value, protecting against unintended state changes.
 
-```rust
+```rust title="tests.rs"
 --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/pallets/custom-pallet/src/tests.rs:83:96'
 ```
 
 ### Origin and Access Control
 
 Confirm that sensitive operations like setting counter value are restricted to authorized origins, preventing unauthorized modifications.
 
-```rust
+```rust title="tests.rs"
 --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/pallets/custom-pallet/src/tests.rs:35:46'
 ```
 
 ### Edge Case Handling
 
 Ensure the pallet gracefully handles edge cases, such as preventing increment operations that would cause overflow.
 
-```rust
+```rust title="tests.rs"
 --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/pallets/custom-pallet/src/tests.rs:98:110'
 ```
 
 ### Verifying State Changes
 
 Test that pallet operations modify the internal state correctly and maintain expected storage values across different interactions.
 
-```rust
+```rust title="tests.rs"
 --8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/pallets/custom-pallet/src/tests.rs:149:164'
 ```
 
 ### Full Test Suite
 
-You can check the complete `tests.rs` implementation for the Custom pallet here:
+You can check the complete `tests.rs` implementation for the custom pallet here:
```

Comment:
I have the same comment here as I previously mentioned - I’d recommend avoiding a leading sentence with a colon when referencing hidden code snippets